src/redux/coinslice.js

```diff
@@ -0,0 +1,28 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+import 'regenerator-runtime/runtime';
+
+/* eslint-disable camelcase */
```

Comment:
- [ ] Kindly consider avoiding disabling the linter because it helps keep code tidy and easier to understand and use throughout the project for anyone who needs to look at the project code in the future